src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
+      transition: 300ms;
```

Comment:
transition shouldn't be on hover, use it directly to the element, because if you move mouse out, transition don't works.
also use transition-property